virttest/qemu_vm.py

```diff
@@ -3147,7 +3159,7 @@ def create(self, name=None, params=None, root_dir=None,
 
             logging.debug("VM appears to be alive with PID %s", self.get_pid())
             # Record vcpu infos in debug log
-            if not self.is_preconfig():
+            if not self.is_preconfig() and not self.is_prelaunch():
```

Comment:
If they are the same thing with different expression in different versions, I would prefer to implement it in one same function, not only because of the meaning, also for the reference of `is_preconfig` at other places.

Reply:
I have the same idea with @vivianQizhu .

The change is only about the qemu monitor output for different qemu version, no change with qemu command:
https://github.com/avocado-framework/avocado-vt/blob/b855e7ec9b2c1f46593d79199668e71f7b92ff3b/virttest/qemu_vm.py#L1514~L1516

I think, `is_preconfig` is still reasonable even if the status string in qemu monitor output is `prelaunch`, so suggest update `is_preconfig` to work with different qemu version.

Reply:
I have the same idea with @vivianQizhu .

The change is only about the qemu monitor output for different qemu version, no change with qemu command:
https://github.com/avocado-framework/avocado-vt/blob/b855e7ec9b2c1f46593d79199668e71f7b92ff3b/virttest/qemu_vm.py#L1514~L1516

I think, `is_preconfig` is still reasonable even if the status string in qemu monitor output is `prelaunch`, so suggest update `is_preconfig` to work with different qemu version.